hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/parsing/PopulateReqHeader.java

```diff
@@ -59,9 +59,8 @@ public void populateReqHeader(Exchange exchange, @Headers Map<String, Object> hm
 			String zcbSegment = V2MessageUtil.getZCBSegment(v2Message,Util.ZCB_SEGMENT);
 			String pharmacyID = V2MessageUtil.getPharmacyId(zcbSegment);
 			String traceID = V2MessageUtil.getTraceNumber(zcbSegment);
-			// These are actual http message headers that get send to Pharmanet
-			hm.put(Util.HEADER_PHARMACY_ID, pharmacyID);
-			hm.put(Util.HEADER_TRACING_ID, traceID);
+			logger.info("{} - Transaction Id : {}, Pharmacy Id : {}, TraceId : {}",
+					methodName, exchange.getExchangeId(), pharmacyID , traceID);
```

Comment:
For consistency, could either use variable here instead of exchange.getExchangeId() or don't use variable at line 43 in ProcessV2ToPharmaNetJson.java and just use exchange.getExchangeId() in the method call also.